src/app/[locale]/(routes)/loans/loan-form.tsx

```diff
@@ -0,0 +1,143 @@
+import { createLoanAction } from '@/app/_action';
+import { TLoan } from '@/models/loan';
+
+const sampleData: TLoan = {
+	borrowerAddress: '0x6B175474E89094C44Da98b954EedeAC495271d0A',
+	collateralAmount: '100',
+	collateralToken: '0x6B175474E89094C44Da98b954EedeAC495271d0B',
+	loanAmount: '100',
+	loanToken: '0x6B175474E89094C44Da98b954EedeAC495271d0C',
+	loanRepayDeadline: new Date(2023 - 12 - 31),
+	initialThreshold: '0.5',
+	interestRate: '0.1',
+	liquidationThreshold: '0.8',
+	lenderAddress: '0x6B175474E89094C44Da98b954EedeAC495271d0D',
+	loanRequestDeadline: new Date(2023 - 12 - 31),
+	managerNFT: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+	managerNFTVersion: 'v1',
+};
+
+const emptyData: TLoan = {
+	borrowerAddress: '',
+	collateralAmount: '',
+	collateralToken: '',
+	loanAmount: '',
+	loanToken: '',
+	loanRepayDeadline: new Date(),
+	initialThreshold: '',
+	interestRate: '',
+	liquidationThreshold: '',
+	lenderAddress: '',
+	loanRequestDeadline: new Date(),
+	managerNFT: '',
+	managerNFTVersion: '',
+};
+
+export default function LoanForm() {
```

Comment:
Put in the readme about this places we are here to test

Why would the metamask not allow you to enter this page?

Reply:
So, a while back in Slack I asked about how I can log in, and what services you are using, but I couldn't find any resources.

Basically, I wanted to know what to do in order to pass the login check, and what services you guys are using/recommend?